src/components/BreadCrums/BreadCrum.js

```diff
@@ -14,7 +14,7 @@ const BreadCrum = () => {
   return (
     <div className="breadcrum">
       <div className="breadcrum_content">
-        <HomeIcon style={{ backgroundColor: "#1e1e1e" }} /> /{" "}
+        <HomeIcon className="breadcrum-home" /> /{" "}
```

Comment:
can we follow the underscore css as it is easier to handle instead of -

Reply:
great catch btw

Reply:
I didn't understand this